locksmith/src/websub/jobs/locks.ts

```diff
@@ -0,0 +1,21 @@
+import { Lock } from '../../graphql/datasource'
+import { Hook } from '../../models'
+import { TOPIC_LOCKS } from '../topics'
+
+import { networkMapToFnResult, notifyHook } from '../helpers'
+
+export async function notifyOfLocks(hooks: Hook[]) {
+  const subscribed = hooks.filter((hook) => {
+    const path = new URL(hook.topic).pathname
+    return TOPIC_LOCKS.test(path)
+  })
+
+  const lockSource = new Lock()
+  const networkToLocksMap = await networkMapToFnResult((network) =>
+    lockSource.getLocks({ first: 25 }, Number(network))
```

Comment:
i think we need to get rid of the 25 limit here?

Reply:
Oh that's a good point actually. Let's keep it like this for now and think about better ways to handle this. I think keep track of a "last time checked" and using it + ⍙ can provide a good mechanism. Wdyt?